db/migrate/20230520010728_create_blogs.rb

```diff
@@ -0,0 +1,17 @@
+class CreateBlogs < ActiveRecord::Migration[6.0]
+  def change
+    create_table :blogs do |t|
+      t.string :title
+      t.string :author_name
+      t.string :author_image
+      t.text :desc
+      t.string :image
+      t.references :author, foreign_key: true
+      t.integer :like_counter, default: 0
```

Comment:
Make sure you edit this column and reference the user table so it will know this author is meant to reference the user table as there is no author table available so as not to bring errors